client/app/private-database/oom/private-database-oom.controller.js

```diff
@@ -11,6 +11,8 @@ angular.module("App").controller(
         }
 
         $onInit () {
+            moment.locale(this.$scope.selectedLanguage.value.split("_")[0]);
```

Comment:
Does the `$scope` always contain a `selectedLanguage` ? And does a locale always contain a `_` ? (I know that in Android, `fr` is a valid locale).

Reply:
I guess so, because that's how we're doin' in every other controller that needs *moment.js*. I'm pretty sure a more elegant solution would require some refactoring though.

Reply:
A `languageService` would be nice I guess _whistles_

Reply:
In general, there is a refactoring to do with all the inherited scopes. 
But this kind of config doesn't belong in a controller, as it can be applicable everywhere in the app.